frontend/express/public/stylesheets/styles/base/_colors.scss

```diff
@@ -23,6 +23,8 @@ $colors: (
     "cool-gray-40": #A7AEB8,
     "cool-gray-30": #CFD6E4,
     "cool-gray-20": #E2E4E8,
+    "cool-gray-10": #F8FAFD,
```

Comment:
actually this is not cool-gray-10.
https://www.figma.com/file/1cRpH4OJ37v6NTDX9Jdrnw/countly-foundations?node-id=132%3A126

Reply:
it must be some other color, please talk to the design team which color to use.
incase its not included in our color map, please let me know.

Reply:
Hello Prikshit I think it is cool-gray only. Took it from figma . Please refer below url. https://www.figma.com/file/PUjfGUiaVQpSBeS5EWMKYu/countly-components?node-id=11161%3A86038(Configure Events Section)

Reply:
but cool-gray-10 in foundations file is some other color 😄

Reply:
wait the foundations file has 2 colors for cool-gray-10 and one of them is yours.
let me confirm with design team.

Reply:
marking this as resolved.
you have added the correct color.